packages/analytics/dashboard-renderer/src/components/DashboardRenderer.vue

```diff
@@ -38,8 +38,16 @@ const props = defineProps<{
 
 const { i18n } = composables.useI18n()
 
+// Note: queryBridge is not directly used by the DashboardRenderer component.  It is required by many of the
+// subcomponents that get rendered in the dashboard, however.  Check for its existence here in order to catch
+// programming errors early.
 const queryBridge = inject(INJECT_QUERY_PROVIDER)
 
+if (!queryBridge) {
+  console.warn('Analytics dashboards require a query bridge supplied via provide / inject.')
+  console.warn("Please ensure your application has a query bridge provided under the key 'analytics-query-provider'")
```

Comment:
suggestion: I'm assuming one of the [public] packages has docs explaining how to configure the query bridge. Can you provide the docs URL in the log?

Reply:
Good thought; done.